test/PHPUnit/Controller/AbstractHttpControllerTestCaseTest.php

```diff
@@ -414,6 +414,18 @@ public function testAssertQueryContentRegex()
         $this->assertQueryContentRegex('div#content', '#o{3,}#');
     }
 
+    public function testAssertQueryContentRegexMultipleMatches()
```

Comment:
Following on the previous comment, please also add a test that validates that the markup I showed above would not pass this assertion.